src/java/org/apache/cassandra/cql3/MultiColumnRelation.java

```diff
@@ -95,6 +96,11 @@ public static MultiColumnRelation createInRelation(List<ColumnIdentifier> entiti
         return new MultiColumnRelation(entities, Operator.IN, null, inValues, null);
     }
 
+    public static MultiColumnRelation createNotInRelation(List<ColumnIdentifier> entities, List<? extends Term.MultiColumnRaw> inValues)
```

Comment:
This could have JavaDoc like the method `createInRelation` right above.

Reply:
Fixed